tests/sentry/api/test_utils.py

```diff
@@ -253,3 +255,30 @@ def test_handle_query_errors(self, mock_parse_error):
                     raise ex
             except Exception as e:
                 assert isinstance(e, (FooBarError, APIException))
+
+
+class IdOrSlugPathParamsEnabledTest(TestCase):
+    def test_no_options_enabled(self):
+        assert not id_or_slug_path_params_enabled("TestEndpoint.convert_args")
+
+    @override_options({"api.id-or-slug-enabled": True})
+    def test_ga_option_enabled(self):
+        assert id_or_slug_path_params_enabled(convert_args_class="TestEndpoint.convert_args")
+
+    @override_options({"api.id-or-slug-enabled-ea-endpoints": ["TestEndpoint.convert_args"]})
+    def test_ea_endpoint_option_enabled(self):
+        assert not id_or_slug_path_params_enabled(convert_args_class="NotTestEndpoint.convert_args")
+        assert id_or_slug_path_params_enabled(convert_args_class="TestEndpoint.convert_args")
+
+    @override_options({"api.id-or-slug-enabled-ea-org": ["sentry"]})
+    def test_ea_org_option_enabled(self):
+        assert not id_or_slug_path_params_enabled("NotTestEndpoint.convert_args", "not-sentry")
+        assert not id_or_slug_path_params_enabled("NotTestEndpoint.convert_args", "sentry")
+
+    @override_options({"api.id-or-slug-enabled-ea-org": ["sentry"]})
+    @override_options({"api.id-or-slug-enabled-ea-endpoints": ["TestEndpoint.convert_args"]})
+    def test_ea_org_and_endpointoption_enabled(self):
```

Comment:
I realized this is missing an underscore lol

Reply:
shoot, I'll fix this in my pr where I actually use this